docs/user-guide/mvd-configuration.md

```diff
@@ -848,17 +915,17 @@ The API returns the following information in a JSON response:
 | API                                                       | Description                                                  |
 | --------------------------------------------------------- | ------------------------------------------------------------ |
 | /server (GET)                                             | Returns a list of accessible server endpoints for the Zowe Application Server. |
-| /server/config (GET)                                      | Returns the Zowe Application Server configuration from the `zluxserver.json` file. |
+| /server/config (GET)                                      | Returns the Zowe Application Server configuration from the `server.json` file. |
```

Comment:
Doesn't this return a merged version of the config from server.json and zowe.yaml?